samcli/commands/local/start_lambda/cli.py

```diff
@@ -49,6 +50,7 @@
     help=HELP_TEXT,
     short_help="Starts a local endpoint you can use to invoke your local Lambda functions.",
 )
+@configuration_option(provider=TomlProvider(cmd="local_start_lambda", section="parameters"))
```

Comment:
Document the "algorithm" to create the command name here.

Reply:
done.